### PR TITLE
MIP102c2-SP17 amendments

### PR DESCRIPTION
**Pull request by blimpa**
MIPs 104, 106, 107, 108, 113 (first articles).

**Do not merge unless RP passes**

### CODE REVIEW

**Review by 0xldr — APPROVED** [review]
We approve this PR as a faithful recreation of the proposed changes through MIP102c2-SP17.

We have noted the github conflicts. We are informed that they shall be resolved at the time of merging, assuming this proposal is ratified by Maker Governance.

**Review by 0xldr — APPROVED** [review]
Approving result of conflict resolution

File: MIP104/MIP104.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ The Stability Facilitators may, if they deem it necessary, trigger a vote to rem
 
 ###### 1.1.1.5.1
 
-Stability Advisory Council members can be individuals, groups of people, legal entities, or companies. They can be pseudonymous or known entities. Stability Advisory Council members must be aligned with the long-term goals of MakerDAO Endgame.
+Stability Advisory Council membAers can be individuals, groups of people, legal entities, or companies. They can be pseudonymous or known entities. Stability Advisory Council members must be aligned with the long-term goals of MakerDAO Endgame.
```
Comment (0xldr) [inline MIP104/MIP104.md]: please fix this prior to merge

```suggestion
Stability Advisory Council members can be individuals, groups of people, legal entities, or companies. They can be pseudonymous or known entities. Stability Advisory Council members must be aligned with the long-term goals of MakerDAO Endgame. 
```

**Comment by blimpa** [comment]
[Vote passed](https://vote.makerdao.com/polling/QmNjTZZm#vote-breakdown)

**Comment by amirkhan7javi** [comment]
It needs to be addressed and the stolen transactions should be investigated and blocked

**Pull request merged**